data/human/human missions.txt

```diff
@@ -3601,3 +3601,79 @@ mission "Mafia Extortion"
+					launch
+				"(Attempt to land anyway.)"
+			"Orbital entry is a bit more steep than normal and your ship is starting to lean a bit."
+			choice
+				"(Try to return to orbit to still fix the issue in space.)"
+					goto reorbit
+				"(Continue the attempt to land.)"
+			"You're reaching the point where your ship will not be able to re-orbit on its own power."
+			"The cockpit is filled with warnings about the descent angle and attitude control."
+			choice
+				"(Attempt to re-orbit and hope that the ship still makes it.)"
+					goto reorbit
+				"(Proceed with landing, crashing, or dying. Whichever one it's gonna be.)"
+			"Your decent is still far more steep than normal, and the ship is starting to roll."
+			"It is no longer possible to correct both your descend angle and maintain attitude control."
+			choice
+				"(Prioritize roll stabilization.)"
+					goto rolling
+				"(Prioritize lowering the descent angle.)"
+				"(Switch to full auto-pilot control.)"
+			"Your ship begins to roll faster and faster, but the autopilot somehow manages to soften the descent angle."
+			choice
+				"(Prioritize roll stabilization now.)"
+					goto rolling
+				"(Continue prioritizing lowering the descent angle.)"
+				"(Let the autopilot handle the landing.)"
+			branch miracle
+				random < 80
+			"It's not enough! The ship starts rolling even faster, your vision blurs, and consciousness starts to slip from you. The last thing you notice is that your ship is falling apart but you already died by the time your ship fully disintegrates."
+				die
+			label rolling
+			"Your ships roll stabilizes, but your descend vector is now almost perpendicular to the surface and the acceleration straight towards the surface pushes you firmly into your chair."
+			choice
+				"(Pray for a miracle.)"
+				"(Scream your lungs out.)"
+				"(Silently accept the situation as it is.)"
+			branch miracle
+				random < 70
+			"There is nothing more you can do to save your ship, your ship crashes straight into the ground. Whoever will have to clean this mess up will have a hard time finding your remains."
+				die
+			label miracle
+			"Suddenly all repulsors come back online at full strength, you feel the seatbelt dig into your skin over the sudden deceleration."
+			choice
+				"(Let the autopilot handle the landing.)"
+				"(Attempt to perform a manual landing.)"
+				"(Try to return to orbit to fix the issue in space.)"
+					goto reorbit
+			branch miraclelanding
+				random < 98
+			"Your ships trajectory and attitude stabilize, but it was too late. All warning-lights in the cockpit are lit and sirens are blaring, but the view out of the window, of the fast approaching planet surface, makes the impending crash more clear than any light or siren does. You experience a few seconds of fear, after which you life ends in a short big flash."
+				die
+			label miraclelanding
+			"Your ships trajectory and attitude stabilize and somehow the ship crash-lands on the surface."
```

Comment:
```suggestion
			"Your ship's trajectory and attitude stabilize, and somehow the ship crash-lands on the surface."
```
To me, a crash landing usually implies damage. Could be worth adding a brief bit more about the player having to pay a few thousand credits to "hammer out dents", "fix the paint job", or something like that - or just clarifying that it's equally miraculous that your ship is somehow undamaged.

Reply:
Good point. And maybe the authorities should also fine the player for landing outside the spaceport.